app/views/lessons/show.html.erb

```diff
@@ -1,3 +1,5 @@
+<% redirect_url = local_assigns.fetch :redirect_url, nil %>
```

Comment:
Does this have any effect? (Since this is a top level page and not a partial, I'm guessing you can't pass it a value like this?

Reply:
It wasn't necessary, good spot! Removed now.

Reply:
Yay!